Jenkinsfile

```diff
@@ -49,8 +49,11 @@ node('docker&&linux') {
     }
 
     stage('Check for typos') {
-      sh 'make check'
-      recordIssues(tools: [checkStyle(id: 'typos', name: 'Typos', pattern: 'checkstyle.xml')], qualityGates: [[threshold: 1, type: 'TOTAL', unstable: true]])
+        if (!infra.isTrusted()) {
```

Comment:
Inconsistent with https://github.com/jenkins-infra/jenkins.io/blob/f9472d3be8b00667c604b7d73941786a659edbc0/Jenkinsfile#L92

Reply:
@daniel-beck does the change in d67b1ba0a9a64548b8e37ae7c6aa61708c8b1bc0 address your concerns enough that this can be merged or are more changes required?